test/k8s-integration/main.go

```diff
@@ -295,12 +312,11 @@ func handle() error {
 		}
 	}
 
-	normalizedVersion, err := getNormalizedVersion(*kubeVersion, *gkeClusterVer)
-	if err != nil {
-		return fmt.Errorf("failed to get cluster minor version: %v", err)
-	}
+	clusterVersion := mustGetKubClusterVersion()
+	fmt.Printf("Kubernetes cluster version %q\n", clusterVersion)
+	testSkip := generateTestSkip(clusterVersion, *useGKEManagedDriver)
```

Comment:
There is a problem in this current logic of unified comparision logic for gce and gke.
For example kubetest --extract=release/latest --up deploys a server version of "v1.19.0-beta.0". Current comparision logic wont work. Needs additional investigation if we can use https://github.com/kubernetes/apimachinery/blob/master/pkg/util/version/version.go for parse and comparision of gke builds. (e.g. v1.19.0-gke.0)